src/utils.rs

```diff
@@ -132,10 +134,62 @@ pub(crate) fn write_wallet_from_mnemonic_and_password(
     .map(|_| ())
 }
 
+/// Ensures there is no wallet at the given [Path], removing an existing wallet if the user has
+/// provided the `--force` option or chooses to remove it in the CLI interaction.
+/// Returns [Err] if there is an existing wallet and the user chooses not to remove it.
+pub(crate) fn ensure_no_wallet_exists(
+    wallet_path: &Path,
+    force: bool,
+    mut reader: impl BufRead,
+) -> Result<()> {
+    if wallet_path.exists() {
+        if force {
+            println_warning(&format!(
+                "Because the `--force` argument was supplied, the wallet at {} will be removed.",
+                wallet_path.display(),
+            ));
+            fs::remove_file(wallet_path).unwrap();
+        } else {
+            println_warning(&format!(
+                "There is an existing wallet at {}. \
+                Do you wish to replace it with a new wallet? (y/N) ",
+                wallet_path.display(),
+            ));
+            let mut need_replace = String::new();
+            reader.read_line(&mut need_replace).unwrap();
+            if need_replace.trim() == "y" {
```

Comment:
```suggestion
            if need_replace.trim().to_lowercase() == "y" {
```

Nothing to block but we may want to cast to lowercase first, so that both `Y` and `y` is accepted.

Reply:
I think it's pretty standard that the user has to match the case to choose the non-default option. Any other keypress is equivalent to selecting the default.